uber/site_sections/preregistration.py

```diff
@@ -453,6 +453,9 @@ def transfer_badge(self, session, message='', **params):
     def confirm(self, session, message='', return_to='confirm', **params):
         attendee = session.attendee(params, bools=_checkboxes, restricted=True)
 
```

Comment:
Hm, that's true. I did it this way because I was thinking that we might ask someone for the URL where they got the message, and I wanted to preserve that URL to make it easy for us.